### ISSUE TITLE
2.0.3 compile error when apk have some file with special character

### ISSUE THREAD

**Issue by nicolas924** [iBotPeaches/Apktool#1154]
I: Using Apktool 2.0.3
I: Smaling smali folder into classes.dex...
[Fatal Error] AndroidManifest.xml:1:1: Content is not allowed in prolog.
I: Copying raw resources...
I: Building apk file...
I: Copying unknown files/dir...
Exception in thread "main" java.lang.ClassCastException: [B cannot be cast to java.lang.String
	at brut.androlib.Androlib.copyUnknownFiles(Androlib.java:635)
	at brut.androlib.Androlib.buildUnknownFiles(Androlib.java:600)
	at brut.androlib.Androlib.build(Androlib.java:307)
	at brut.androlib.Androlib.build(Androlib.java:268)
	at brut.apktool.Main.cmdBuild(Main.java:225)
	at brut.apktool.Main.main(Main.java:84)

**Comment by nicolas924** [comment]
if on windows it displayed error like #1129

**Comment by iBotPeaches** [comment]
Do you have an apk by chance?

**Comment by nicolas924** [comment]
Can I email it to you ?

**Comment by iBotPeaches** [comment]
Sure. ibotpeaches (at) gmail (dot) com

I'll keep it private, if thats why you are choosing to email.

**Comment by nicolas924** [comment]
thankyou very much!

2016-02-22 0:12 GMT+07:00 Connor Tumbleson <notifications@github.com>:

**Comment by iBotPeaches** [comment]
```
➜  Bug1154 [redacted].apk -f
I: Using Apktool 2.1.0-1ff3a3-SNAPSHOT on [redacted].apk
I: Loading resource table...
I: Decoding AndroidManifest.xml with resources...
I: Loading resource table from file: /home/ibotpeaches/apktool/framework/1.apk
I: Regular manifest package...
I: Decoding file-resources...
I: Decoding values */* XMLs...
I: Baksmaling classes.dex...
I: Copying assets and libs...
I: Copying unknown files...
I: Copying original files...
a%                                                                                                                                                                                 ➜  Bug1154 apktool b [redacted]
I: Using Apktool 2.1.0-1ff3a3-SNAPSHOT
I: Checking whether sources has changed...
I: Smaling smali folder into classes.dex...
I: Checking whether resources has changed...
I: Building resources...
I: Building apk file...
I: Copying unknown files/dir...
➜  Bug1154 
```

I'd check your computer charsets or maybe this has already been fixed in the upcoming 2.1.0 release.
